NetKAN.schema

```diff
@@ -42,39 +42,39 @@
             "type": "string",
             "oneOf": [
                 {
-                    "pattern": "^#/ckan/spacedock/[0-9]+",
+                    "pattern": "^#\/ckan\/spacedock\/[0-9]+$",
```

Comment:
```suggestion
                    "pattern": "^#/ckan/spacedock/[0-9]+$",
```

I did look up the backslash forward slash thing before submitting the previous PR, and it seems to not be needed, since unlike with `sed` or Perl, there are no slashes around the outside of the regex.

- <https://stackoverflow.com/questions/1580647/json-why-are-forward-slashes-escaped>

I confirmed that by testing at <https://www.jsonschemavalidator.net/>.

Let's revert these to single forward slashes for concision and ease of reading (everywhere).

Reply:
I based my superfluous backslashes entirely on https://regex101.com/ reporting faulty regex for JS (but it does work in VSCode anyway 🤷‍♂️), these would just make it consumable for more regex engines without change.

But this is not an important consideration and was just personal taste, so I will gladly defer to your pick!